pyrandwalk/pyrandwalk_obj.py

```diff
@@ -84,20 +84,30 @@ def prob_sec(self, sequence, initial_dist=None):
             current_state = next_state
         return probability
 
-    def run(self, ntimes=10, show=False):
+    def run(self, ntimes:int=10, show:bool=False, starting_state:int=None):
         """
         Run random walk for ntimes and print out result on each transition.
 
         :param ntimes: numbers of running
         :type ntimes: int
         :param show: flag which is set when showing run is desired
+        :starting_state: the state from which the randon walk should start from
         :return: (generated_states, probability)
         """
-        state = np.random.choice(self.S)
+        if starting_state == None:
+            state = np.random.choice(self.S)
+        else:
+            state = starting_state
+            
```

Comment:
It's better to write it like the below:
```
state = np.random.choice(self.S)
if starting_state != None:
    state = starting_state
```

Reply:
Sure, sounds good. I wrote it with if else for better readability, but will change it :)

Reply:
I personally believe that more indented lines will make the code less readable. This stereotype for optional arguments is what we use in our other projects, too.